packages/celotool/src/e2e-tests/sync_tests.ts

```diff
@@ -28,18 +36,54 @@ describe('sync tests', function(this: any) {
     await hooks.before()
     // Restart validator nodes.
     await hooks.restart()
+    const validatorKit = newKitFromWeb3(new Web3('http://localhost:8545'))
+    const groupPrivateKey = await getValidatorGroupPrivateKey(validatorKit, gethConfig.instances[0])
     const fullInstance = {
       name: 'full',
       validating: false,
       syncmode: 'full',
       lightserv: true,
-      port: 30311,
-      rpcport: 8553,
+      port: 30315,
+      wsport: 8557,
+      rpcport: 8559,
+      privateKey: groupPrivateKey.slice(2),
       peers: [8545],
     }
     await initAndStartGeth(hooks.gethBinaryPath, fullInstance)
-    const web3 = new Web3('http://localhost:8553')
+    const validators = await (await validatorKit._web3Contracts.getValidators()).methods
+      .getRegisteredValidators()
+      .call()
+    const web3 = new Web3('ws://localhost:8557')
+    await sleep(10)
     await waitToFinishSyncing(web3)
+    const kit = newKitFromWeb3(web3)
+    // The validator set size at any one time will be 4, and we rotate two validators per epoch.
+    // This means that if a node were to have a static view of the validator set, we guarantee that
+    // the node will reject blocks.
```

Comment:
Maybe add reference or link to the celo-blockchain code that will reject the block due to differing validator set?

Reply:
Explicitly pointed out the function that would return false